### DISCUSSION TITLE
Emcee import error

### DISCUSSION THREAD

**Discussion by kathmcp** [phoebe-project/phoebe2#511] [Questions]
Hi again Kyle, so thanks to your help, I have managed to get my optimizer working :) However, I am now getting an import error when I try to run the emcee solver. Have I missed a line of code to import the solver? I have been following the "Advanced: Emcee Solver" tutorial and "Inverse Problem: General Workflow and Examples". Thanks!

**Comment by aprsa** [comment]
Just checking the most obvious: did you `pip install emcee`?

**Reply by kathmcp** [reply]
I hadn't, but have now. I did wonder if I should, but thought I better check first! Thank you :)
